core/storage/trie_pruner/impl/trie_pruner_impl.cpp

```diff
@@ -380,14 +360,12 @@ namespace kagome::storage::trie_pruner {
     while (!queued_nodes.empty()) {
       auto [node, hash] = queued_nodes.back();
       queued_nodes.pop_back();
-      SL_TRACE(logger_,
-               "Add - Node {}, ref count {}",
-               hash.toHex(),
-               ref_count_[hash]);
+      auto ref_count = ref_count_[hash];
+      SL_TRACE(logger_, "Add - Node {}, ref count {}", hash.toHex(), ref_count);
```

Comment:
If short hash is enough for log, we can do such change:
```suggestion
      SL_TRACE(logger_, "Add - Node {}, ref count {}", hash, ref_count);
```